pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -22,6 +22,8 @@ in buildFHSUserEnv rec {
     xorg.libICE
     xorg.libSM
     zlib
+    dbus
+    libxcrypt
```

Comment:
libxcrypt is already there in master branch. Does this PR need a rebase, perhaps? (And update the commit message.)